app/src/androidTest/java/com/github/houseorganizer/houseorganizer/LoginActivityTest.java

```diff
@@ -55,4 +65,14 @@ public void discoverButtonIsEnabled() {
         onView(withId(R.id.discoverButton)).check(matches(isEnabled()));
     }
 
+    @Test
+    public void backPressLeavesApp() {
+        // Closing the app throws NoActivityResumedException, so we make the test fail if nothing was thrown
+        Intents.init();
+        try {
+            pressBack();
+            fail("Should have thrown NoActivityResumedException");
+        } catch (NoActivityResumedException expected) { }
+        Intents.release();
```

Comment:
Isn't it better to put the realease in a finally block ? I am not sure how it works though

Reply:
actually i think the safest way was to put them in the @after and the Intents.init() in the @before, ill move them there